app/controllers/TimeToPayController.scala

```diff
@@ -24,27 +24,24 @@ import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.LoggerUtil
 
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class TimeToPayController @Inject()(authorisedController: AuthorisedController,
                                     mcc: MessagesControllerComponents,
                                     timeToPayService: TimeToPayService,
                                     errorHandler: ServiceErrorHandler)
-                                   (implicit ec: ExecutionContext,
-                                    appConfig: AppConfig)
+                                   (implicit ec: ExecutionContext, appConfig: AppConfig)
   extends FrontendController(mcc) with I18nSupport with LoggerUtil {
 
   def redirect: Action[AnyContent] = authorisedController.authorisedAction { implicit request => _ =>
-    if(appConfig.features.overdueTimeToPayDescriptionEnabled()) {
-      timeToPayService.retrieveRedirectUrl.map {
-        case Right(url) => Redirect(url)
-        case Left(_) =>
-          logger.warn("[TimeToPayController][redirect] - Unable to retrieve successful response from TTP service, rendering ISE")
-          errorHandler.showInternalServerError
-      }
-    } else {
-      Future.successful(NotFound(errorHandler.notFoundTemplate))
+    timeToPayService.retrieveRedirectUrl.map {
+      case Right(url) => Redirect(url)
```

Comment:
![image](https://github.com/user-attachments/assets/da2394a0-ca4f-45a2-8111-2d310718daa2)

![image](https://github.com/user-attachments/assets/bd8d746b-7091-4018-98f7-bd1e12201994)


Please include a screen shot for charge type journey

Reply:
![image](https://github.com/user-attachments/assets/6e38123a-ecde-4b4d-92fc-ad5c11e67107)
the "Payment Help" link is present along with what comes up when the link is clicked